Mlem/App State.swift

```diff
@@ -27,4 +26,21 @@ class AppState: ObservableObject {
     @Published var toast: AlertToast?
 
     @Published var criticalErrorType: CriticalError = .shittyInternet
+
+    @Published var enableDownvote: Bool = true
+    
+    func setActiveAccount(_ account: SavedAccount?) {
+        currentActiveAccount = account
+        
+        if let newAccount = account {
+            Task {
+                let request = GetSiteRequest(account: newAccount)
+                if let response = try? await APIClient().perform(request: request) {
+                    await MainActor.run {
+                        enableDownvote = response.siteView.localSite.enableDownvotes
```

Comment:
After talking about how we will need language data I wonder if we can just cache the entire site view.

Reply:
We should definitely store them here once we know how we're going to use them, to save doing a separate call 👍